src/chainparams.cpp

```diff
@@ -40,12 +40,12 @@ static CBlock CreateGenesisBlock(const char* pszTimestamp, const CScript& genesi
     return genesis;
 }
 
-static CBlock CreateDevNetGenesisBlock(const uint256 &prevBlockHash, const std::string& devNetName, uint32_t nTime, uint32_t nNonce, uint32_t nBits, int32_t nVersion, const CAmount& genesisReward)
+static CBlock CreateDevNetGenesisBlock(const uint256 &prevBlockHash, const std::string& devNetName, uint32_t nTime, uint32_t nNonce, uint32_t nBits, const CAmount& genesisReward)
 {
     assert(!devNetName.empty());
 
     CMutableTransaction txNew;
-    txNew.nVersion = 4;
+    txNew.nVersion = 2;
```

Comment:
nit: v1 should be enough here

Reply:
v2 started to use the height in coinbase and I wanted to provide the height in the genesis block as well. don't remember however why I wanted this in the first place

Reply:
I think you are confusing tx/block version here again then - block v2 (bip34) started to use height, tx v2 is about bip68, not bip34.

Reply:
Yeah, you're right. This confusion seems to haunt me...adding a commit on top